mreg_cli/commands/host_submodules/core.py

```diff
@@ -255,9 +277,11 @@ def forced(override_required: str | None = None) -> bool:
         return False
 
     warnings: list[str] = []
+    overrides_required: set[str] = set()
     # Require force if host has any cnames.
-    if host.cnames and not args.force:
-        warnings.append(f"  {len(host.cnames)} cnames, override with 'cname'")
+    if host.cnames and not forced(Override.CNAME):
+        overrides_required.add(Override.CNAME)
+        warnings.append(f"  {len(host.cnames)} cnames")
         for cname in host.cnames:
```

Comment:
Require `cname` override or not? Can remove it if we want to treat it as a special case alongside the "multiple addresses on same VLAN" case, where we don't require an override, only force.

Reply:
Override on `cname` is probably overkill, but it is consistent and now self-documenting, so I think it's fine to have it.